test/common.py

```diff
@@ -83,6 +83,11 @@
 if 'EM_BUILD_VERBOSE' in os.environ:
   exit_with_error('EM_BUILD_VERBOSE has been renamed to EMTEST_BUILD_VERBOSE')
 
+# If we are drawing a parallel swimlane graph of test output, we need to use a temp
+# file to track which tests were flaky so they can be graphed in orange color to
+# visually stand out.
+EMTEST_FLAKY_TEST_LOG_FILE = os.path.join(tempfile.gettempdir(), 'emscripten_flaky_tests') if os.getenv('EMTEST_VISUALIZE') else None
```

Comment:
No need for `EMTEST_` prefix here.

Also, since you aren't actually opening the file here maybe just `flaky_test_log_file = os.path.join(tempfile.gettempdir(), 'emscripten_flaky_tests')`  (i'm not a fan on the single line if/else syntax and it seems like we can avoid it here.)

Also, maybe give the file the `.txt` externsion?

Also, why not just write to `out/flaky_texts.txt`?  (why use tmpdir at all?)

Reply:
ok, changed to `out/flaky_tests.txt`